app/components/dialogs/settings/SettingsDialog.tsx

```diff
@@ -227,6 +231,28 @@ const SettingsDialog = (props: Props) => {
         />
       )}
 
+      {isResetSettingsDialogOpened && (
+        <ConfirmDialog
+          open={isResetSettingsDialogOpened}
+          onClose={() => {
+            setIsResetSettingsDialogOpened(false);
+          }}
+          title="Confirm"
+          content={i18n.t('core:confirmResetSettings')}
+          confirmCallback={result => {
+            if (result) {
+              const electron = window.require('electron');
+              const webContents = electron.remote.getCurrentWebContents();
+              webContents.session.clearStorageData();
```

Comment:
why not localStorage.clear(); https://developer.mozilla.org/en-US/docs/Web/API/Storage/clear

Reply:
localStorage will be undefined in Electron app